### PR TITLE
Add elokoyi to people

### PR DESCRIPTION
**Pull request by elokoyi1989**

### CODE REVIEW

File: about/people.yml [hunk 1]
```diff
@@ -483,3 +483,14 @@ data:
...
     image: austin_main.jpg
+  - name: Elizabeth Lokoyi
+    type: Part Time
+    team: Advanced Research Computing
+    subteam: Computational Biology Core
+    title: Web Development Intern
+    github_username: elokoyi1989
+    brown_directory_uuid: 00cdb94d-c25e-4df2-8128-95ef4c135721
+    bio: Elizabeth is a rising senior at Brown University studying Computer Science and English working as a web deelopment intern. Her interests include web development, design, game development, and film making. On her free time, you can find her trying out new cafes!
+    image: elizabeth_main.jpg
```
Comment (eldu) [inline about/people.yml:494]: this image name needs to match your image filename!

File: images/people/Elizabeth_hover.png
Comment (eldu) [inline images/people/Elizabeth_hover.png:1]: Need to convert this image to a `.jpg`

File: images/people/Elizabeth_main.jpeg
Comment (eldu) [inline images/people/Elizabeth_main.jpeg:1]: Also need to convert this to a `.jpg`

**Review by eldu — CHANGES_REQUESTED** [review]
The images in the preview aren't showing up! Love your bio!!! 

A few changes to go :)

**Review by eldu — DISMISSED** [review]
Ah, looks like your images are still not showing up in the preview. 

In your image names, you have `Elizabeth_main.jpg` and `Elizabeth_hover.jpg`. You'll need to make that Elizabeth lowercase to match what you have in the yaml file

So close !!!

**Review by elokoyi1989 — COMMENTED** [review]
I believe I have fixed all the resuested changes fingers crossed

**Review by eldu — APPROVED** [review]
It looks good to me! Let's merge it in :)

**Pull request merged**